qutebrowser/config/websettings.py

```diff
@@ -238,6 +238,25 @@ def _set(self, value, qws=None):
         self._setter(*args)
 
 
+class ThirdPartyCookies(Base):
+
+    """The ThirdPartyCookiePolicy setting is different from other settings."""
+
+    mapping = (
+        ('always', QWebSettings.AlwaysAllowThirdPartyCookies),
+        ('never', QWebSettings.AlwaysBlockThirdPartyCookies),
+        ('existing', QWebSettings.AllowThirdPartyWithExistingCookies),
+    )
+
+    def get(self, qws=None):
+        policy = QWebSettings.globalSettings().thirdPartyCookiePolicy()
+        return tuple(filter(lambda i: i[1] == policy, self.mapping))[0][0]
+
+    def _set(self, value, qws=None):
+        x = filter(lambda i: i[0] == value, self.mapping)
+        QWebSettings.globalSettings().setThirdPartyCookiePolicy(tuple(x)[0][1])
```

Comment:
I *think* I know what this does after staring at it for a while, but this can definitely be done *much* easier: Make `mapping` a dict and use `self.mapping[value]`.